keda/values.yaml

```diff
@@ -89,15 +89,21 @@ logging:
     level: 0
 
 podSecurityContext: {}
-  # fsGroup: 2000
+  # operator:
+  #   fsGroup: 2000
+  # metricServer:
+  #   fsGroup: 2000
 
 securityContext: {}
-  # capabilities:
-  #   drop:
-  #   - ALL
```

Comment:
Please add this to the docs of the new flavors

Reply:
I will, but I thought they don't make much sense with runAsNonRoot: true

Reply:
Added